scripts/in_container/run_check_default_configuration.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import os
+import subprocess
+import tempfile
+
+default_config_cmd = [
+    "airflow",
+    "config",
+    "list",
+    "--default",
+]
+check_lint_cmd = [
+    "airflow",
+    "config",
+    "lint",
+]
+expected_output = "No issues found in your airflow.cfg."
+
+if __name__ == "__main__":
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        # Write default config cmd output to a temporary file
+        default_config_file = os.path.join(tmp_dir, "airflow.cfg")
+        with open(default_config_file, "w") as f:
+            result = subprocess.run(default_config_cmd, stdout=f)
+        if result.returncode != 0:
+            print("\033[0;31mERROR: when running `airflow config list`\033[0m\n")
```

Comment:
this string can be refactored as a constant to avoid


```suggestion
            print("\033[0;31mERROR: when running `airflow config lint`\033[0m\n")
```